packages/components/app/styles/components/pagination.scss

```diff
@@ -10,19 +10,44 @@
 
 .hds-pagination {
   display: grid;
-  grid-template-areas: "left center right";
+  grid-template-areas: "info nav selector";
   grid-template-rows: auto;
   grid-template-columns: 1fr auto 1fr;
   align-items: center;
   margin: 0 auto;
 }
 
+// Responsive layout
+// max-width is based on where component layout potentially breaks down
+@media screen and (max-width: 1000px) {
```

Comment:
@KristinLBradley have you tried inverting the approach, using `flexbox` by default and switching to a grid above `1000px`? I _think_ the overall code will simplify quite a bit, without the need to override so many properties.

Reply:
Ok, I'll look into that...

Reply:
I cleaned up some of the code a bit but I want to wait until we decide which layout we are going with before I do further optimizations.

Reply:
I broke up the responsive styles and nested them with styles for the elements they affect. To me, it didn't look like it would simplify the code to use flexbox by default (I cleaned up and simplified the styles since I first opened this PR though.)